### ISSUE TITLE
[QUESTION/][Floppy] Problems with setting environment variable

### ISSUE THREAD

**Issue by naurias** [manilarome/the-glorious-dotfiles#119]
In section [Rice Qt5 apps](https://github.com/manilarome/the-glorious-dotfiles/wiki/How-Tos#rice-qt5-apps-on-non-plasma-environment) on 2nd instruction it says that set environment variable in something like xprofile. There isn't any xprofile in my home directory and i don't know what other files can be used for setting the environment variable. Should i create a new file (xprofile) in home or find it somewhere (i can't find xprofile anywhere). I'm using lightdm (gtk greeter) and lxdm

All I could find was 
/etc/xdg/xfce4/xfconf/xfce-perchannel-xml/xsettings.xml
and 
/etc/profile (text file)

I don't know what I'm saying 

in short I don't know where to set environment variables

Thanks for such awesome configs tho. Loving it so far

**Comment by ArtemSmaznov** [comment]
In short yes,  you just create the file manually if it doesn't exists in your home directory.
I personally declare my env variables in .bashrc in my home directory

Arch Wiki is quite helpful with these kinds of things so I would recommend to read up on it.
https://wiki.archlinux.org/index.php/Environment_variables#Graphical_environment
and
https://wiki.archlinux.org/index.php/Xprofile

**Comment by manilarome** [comment]
Just like @ArtemSmaznov said, you can just create it or put your env vars in your bashrc.
